packages/server/graphql/public/typeDefs/RecurrenceSettingsInput.graphql

```diff
@@ -0,0 +1,14 @@
+"""
```

Comment:
+1 Should this just be in `updateRecurrenceSettings.graphql`? I think I've seen these sorts of mutation inputs defined next to the mutation, similar to the return payload.

Reply:
we need it in two places - `startTeamPrompt` and `updateRecurrenceSettings` so I figured it'd be best to have it in separate file, but I do have a little tension related to this. maybe we should have two separate input types defined in each mutation as they may start to diverge?

Reply:
Ahh gotcha, yea this should be in it's own file then. Though perhaps we rename it to just `RecurrenceSettings` or similar? Calling it `RecurrenceSettingsInput`, and then using it as an argument to `startTeamPrompt` and `updateRecurrenceSettings` is maybe a bit odd.

Reply:
At this point I'm fairly certain we should use separate types (even though they'll look the same for now). I used `Input` in the filename to avoid confusion with normal gql type. this one is just for mutation inputs. @mattkrick do you have any thoughts here?